addressableled/robot.py

```diff
@@ -0,0 +1,50 @@
+from wpilib import run, TimedRobot, AddressableLED
```

Comment:
We avoid using `from wpilib import ...` in our examples, as it's clearer to refer to things in the top level (such as `wpilib.run`) via the module name. Doing this also makes it easier to build on the examples.

Reply:
okay!